src/utils/text.ts

```diff
@@ -0,0 +1,2 @@
+export const breakStringWithQuotesBySpaces = (string: string) =>
+  string.match(/("[^"]*?"|[^"\s]+)+(?=\s*|\s*$)/g); // source: https://stackoverflow.com/a/16261693/1265472
```

Comment:
Adds explicit return type + reformats comment
```suggestion
/**
 * @see https://stackoverflow.com/a/16261693/1265472
 */
export const breakStringWithQuotesBySpaces = (
  string: string
): RegExpMatchArray | null => string.match(/("[^"]*?"|[^"\s]+)+(?=\s*|\s*$)/g);
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1460/commits/dd079984f7ece5b6a3ab9c8a974b1ab0d92d8c5a